packages/styles/index.html

```diff
@@ -4,33 +4,47 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Styles playground</title>
-    <link id="tokens" rel="stylesheet" href="src/post-tokens-external.scss" />
+
+    <!-- Token imports -->
+    <link rel="stylesheet" href="src/tokens/_core.scss" />
+    <link id="channel" rel="stylesheet" href="src/tokens/_external.scss" />
+    <link id="palettes" rel="stylesheet" href="src/palettes/post-palettes.scss" />
+    <link id="theme" rel="stylesheet" href="src/tokens/_post-theme.scss" />
+    <link rel="stylesheet" href="src/tokens/_device.scss" />
+    <link rel="stylesheet" href="src/tokens/_components.scss" />
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="src/layout/index.scss" />
+    <link rel="stylesheet" href="src/utilities/_index.scss" />
     <link rel="stylesheet" href="src/elements/_index.scss" />
     <link rel="stylesheet" href="src/components/_index.scss" />
+    <script src="playground.js" defer></script>
   </head>
-  <body class="p-4">
-    <h1>Styles package playground</h1>
-    <p class="mb-4">
-      Use this playground for quickly developing HTML/CSS components. Run
-      <code style="background-color: lightgrey; padding: 0.125rem 0.25rem; border-radius: 0.25rem"
-        >pnpm styles:start</code
-      >
-      to get going.
-    </p>
-    <label for="tokens-select">Select tokens package: </label>
-    <select
-      name="tokens"
-      id="tokens-select"
-      value="external"
-      class="mb-4"
-      onchange="document.getElementById('tokens').setAttribute('href', `/src/post-tokens-${this.value}.scss`)"
-    >
-      <option value="external">Post external</option>
-      <option value="internal">Post internal</option>
-    </select>
-    <hr />
+  <body class="palette-default">
```

Comment:
You don't need to set a palette on the body: 
```suggestion
  <body>
```